pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -57,6 +57,58 @@ sb.append('a');     // use this instead
         </example>
     </rule>
 
+    <rule name="AvoidApacheCommonsFileItemNonStreaming"
+          since="6.17.0"
+          language="java"
+          message="Avoid memory intensive FileItem.get and FileItem.getString"
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          typeResolution="true"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_performance.html#avoidapachecommonsfileitemnonstreaming">
+        <description>
+            Problem: Use of FileItem.get and FileItem.getString could exhaust memory since they load the entire file into memory&#13;
+            Solution: Use streaming methods and buffering.
```

Comment:
Could you mention `getInputStream()` explicitly in the description? Like for the name of the rule, the "streaming methods" here sounds unclear to me